exdb/forms.py

```diff
@@ -0,0 +1,69 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.forms import ModelForm
+from django.utils.timezone import now
+from exdb.models import Experience
+
+
+class ExperienceSaveForm(ModelForm):
+
+    class Meta:
+        model = Experience
+        fields = ['name', 'description', 'planners', 'start_datetime', 'end_datetime', 'type',
+                  'sub_type', 'recognition', 'audience', 'attendance', 'keywords', 'goal', 'guest',
+                  'guest_office']
+        
+        widgets = {
+            'description': forms.Textarea(attrs={'cols': 40, 'rows': 4}),
+            'goal': forms.Textarea(attrs={'cols': 40, 'rows': 4}),
+            'start_datetime': forms.SelectDateWidget(),
+            'end_datetime': forms.SelectDateWidget(),
+        }
+
+        labels = {
+            'start_datetime': 'Start Time',
+            'end_datetime': 'End Time',
+        }
+
+    def __init__(self, *args, **kwargs):
+        self.when = kwargs.pop('when', now())
+        return super(ExperienceSaveForm, self).__init__(*args, **kwargs)
+
+
+class ExperienceSubmitForm(ExperienceSaveForm):
+    def clean(self):
+
+        if not self.cleaned_data.get('end_datetime'):
+            raise ValidationError("Need a end date!")
```

Comment:
"date" should be changed to "time" so that it's more consistent with the field label ("End Time").